doctr/local.py

```diff
@@ -248,3 +248,20 @@ def check_repo_exists(deploy_repo, service='github', *, auth=None, headers=None)
     r.raise_for_status()
 
     return r.json().get('private', False)
+
+def in_git_root():
+    """
+    Check if current working directory is root dir of git repo.
+    Return ``False`` if it isn't.
+    """
+    repo = subprocess.run(['git', 'rev-parse', '--show-toplevel'],
```

Comment:
My thought was that someone _might_ want to run `doctr` in a subdirectory for some specific reason? I can't think of what that would be, but that was the thought.